tempesta_fw/sock.c

```diff
@@ -364,9 +365,12 @@ ss_do_send(struct sock *sk, struct sk_buff **skb_head, int flags)
 
 		ss_skb_init_for_xmit(skb);
 
-		TFW_DBG3("[%d]: %s: entail skb=%p data_len=%u len=%u\n",
-		         smp_processor_id(), __func__,
-		         skb, skb->data_len, skb->len);
+		/* Propagate mark of message head skb.*/
+		skb->mark = mark;
```

Comment:
Please extend #883 by a requirement to test the mark iteroperability between Tempesta FW and iptables/nftables in both the directions (or just write the test on your own). Also add an example with iptables/nftables to https://github.com/tempesta-tech/tempesta/wiki/HTTP-tables

Reply:
Wiki has been updated with iptables example.